### DISCUSSION TITLE
Timeout when running ‘pytest tests/test_bokehjs.py’

### DISCUSSION THREAD

**Discussion by muendlein** [bokeh/bokeh#14101] [Build / CI]
It seems like running `pytest tests/test_bokehjs.py` will use the 90s timeout specified inside pyproject.toml. However this is not sufficient therefore triggering a timeout during execution (in my case taking around 110-150s, tested on Windows + MacOS)
Note: Running the same set of tests via `node make test` works fine (I guess the timeout setting is not relevant here) and the CI is obviously also not having those issues (is `pytest tests/test_bokehjs.py` even used there?).

**Comment by mattpap** [comment]
This timeout is a generic one for any test or test suite (not sure which one exactly). 90s is definitely not enough time this days to run bokehjs' tests. We will have to increase the time limit for this particular test. Though, I would actually prefer to remove this test file, to avoid having multiple ways of doing things. Especially when one way (`node make test`) is significantly better and the other (e.g. you don't have to wait for all tests to finish to see results). `test_bokehjs.py` is not used in CI.
